### ISSUE TITLE
java.lang.reflect.InvocationTargetException:  Null error on modpack load

### ISSUE THREAD

**Issue by Toa-Sampa** [DrMango14/Create-The_Factory_Must_Grow#23]
so, creating a modpack with a lot of Create sub-mods in 1.18.2.  I have attempted multiple times to use this, but everytime, even after the update, I get the above error.  I have provided the crash report to help as well as a screenshot provided.

![2023-12-11_16 06 19](https://github.com/DrMango14/Create-The_Factory_Must_Grow/assets/67250785/b0b9d56e-ae5c-4e4a-b1e6-3ce14753d3fe)

[crash-2023-12-11_16.02.40-fml.txt](https://github.com/DrMango14/Create-The_Factory_Must_Grow/files/13632059/crash-2023-12-11_16.02.40-fml.txt)

**Comment by TechSupportRed** [comment]
same issue here

**Comment by Artex112** [comment]
Yup, same here no idea why it happens. it may be something with flywheel but idk.

**Comment by CSdiscoveries** [comment]
Same issue, can't figure it out. I'm using the latest version of everything related, the only thing crashing my game (forge, 1.18.2) is when I add the latest appropriate version of this mod to the folder. Otherwise there are no issues.

**Comment by youngikeson** [comment]
same here. tho i this i can big more info. it pure create-1.18.2-0.5.1.f and createindustry-0.5.2f-1.18.2-yeehaw. well mine is for this way. so i have no dont it also for the rest of you too. this probably do to the lastest version of create the factory must grow is from 7/6/2023 and the lastest create is from 11/2/2023 so most like the new create changed somthing and it brakes create the factory must grow. it may seem like a simple fix by just getting to be load correctly with the newer create, but it probably now plus i dont now any of the code so it may be hard is may take long or it can be quick and easy. it best for the mod team to get more exact information on this issue. but i probaly be fixed new update for 1.18.2 if we lucky and if there will be another update.
[crash-2024-01-27_15.30.45-fml.txt](https://github.com/DrMango14/Create-The_Factory_Must_Grow/files/14073803/crash-2024-01-27_15.30.45-fml.txt)
[latest.log](https://github.com/DrMango14/Create-The_Factory_Must_Grow/files/14073804/latest.log)
![image_2024-01-27_154137503](https://github.com/DrMango14/Create-The_Factory_Must_Grow/assets/97610657/b9e77a5e-c0d3-4ba4-a203-f4012e851cf2)

**Comment by deepslater** [comment]
Hey guys, the issue is that TFMG is not updated to support the latest version of Create on 1.18.2. I believe the last version supported is Create v0.5.1c, but I can't confirm right now. TFMG doesn't seem that well-maintained, so I would expect for this issue to not be solved. You can downgrade Create to the version I mentioned, but then you'll also need to downgrade ALL your other dependents to the version that uses v.0.5.1c. Good luck

**Comment by LIPDProductionsInc** [comment]
Honestly, it's starting to look like this project is abandoned now. Which sucks cause I already spent resources with this mod including some factories...

If anyone is willing to make a pull request and fix this, please let us know as there's a bunch of other add-ons I have to turn down because of this one

**Comment by DrMango14** [comment]
1.18.2 no longer supported
